lms/static/certificates/sass/_components.scss

```diff
@@ -0,0 +1,641 @@
+            color: palette(grayscale, x-dark);
+        }
+    }
+
+    .recipient-img, .recipient-details {
+        display: inline-block;
+        vertical-align: middle;
+    }
+
+    .recipient-img {
+        @extend %img-cropped;
+        width: rem(100);
+        margin-right: spacing-horizontal(small);
+
+        .src {
+            display: block;
+            max-height: 100%;
+            border: rem(4) solid palette(grayscale, white-t);
+        }
+    }
+
+    .recipient-details {
+        @extend %copy-base;
+        font-weight: font-weight(semi-bold);
+    }
+
+    .recipient-name {
+        margin-bottom: spacing-vertical(xx-small);
+    }
+
+    .recipient-username {
+        margin-bottom: 0;
+    }
+
+    .accomplishment-id .value {
+        word-wrap: break-word;
+    }
+}
+
+.accomplishment-metadata-title {
+    @extend %hd-subsection;
+    border-bottom-color: palette(grayscale-cool, light);
+}
+
+// ------------------------------
+// #ACCOMPLISHMENT RELATED
+// ------------------------------
+.accomplishment-related {
+
+}
+
+.accomplishment-related-title {
+    @extend %hd-subsection;
+}
+
+// ------------------------------
+// #CERTIFICATE
+// ------------------------------
+// certficate - basic
+.certificate {
+
+    .accomplishment-rendering {
+        border-top-color: $cert-base-color;
+
+        // decorative corners for certs
+        .deco-corner {
+            position: absolute;
+        }
+
+        .deco-corner-tl {
+            @include triangle(20px, $cert-decorative-color-de-emphasized, up-right);
+            top: 0;
+            right: 0;
+        }
+
+        .deco-corner-tr {
+            @include triangle(20px, $cert-decorative-color-de-emphasized, up-left);
+            top: 0;
+            left: 0;
+        }
+
+        .deco-corner-bl {
+            @include triangle(20px, $cert-decorative-color-de-emphasized, down-right);
+            bottom: 0;
+            right: 0;
+        }
+
+        .deco-corner-br {
+            @include triangle(20px, $cert-decorative-color-de-emphasized, down-left);
+            bottom: 0;
+            left: 0;
+        }
+
+        .accomplishment-type-label {
+            color: $cert-base-color;
+        }
+
+        .accomplishment-signatories-title, .accomplishment-orgs-title {
+            color: $cert-decorative-color;
+        }
+
+        // statement
+        .accomplishment-statement,
+        .accomplishment-statement-lead,
+        .accomplishment-recipient,
+        .accomplishment-course,
+         {
+            font-family: $font-family-serif;
+        }
+
+        .accomplishment-recipient,
+        .accomplishment-course {
+            letter-spacing: rem(1);
+        }
+    }
+}
+
+// certificate - distinguished
+.certificate-base {
+
+    .introduction {
+        margin-bottom: 0;
+    }
+
+    .accomplishment-main {
+        background: transparentize($cert-base-color, 0.9);
+    }
+
+    .accomplishment-rendering {
+
+        // hide the fancy
```

Comment:
NOTE: Poor man's way of visually differentiating base vs. distinguished. The DOM items noted here should be removed from the template rendering as well.